python/graphstorm/config/argument.py

```diff
@@ -733,7 +733,9 @@ def eval_batch_size(self):
         if hasattr(self, "_eval_batch_size"):
             assert self._eval_batch_size > 0
             return self._eval_batch_size
-        return self.batch_size
+        # (Israt): This number is heuristically chosen. Benchmarks can be found at PR:
+        # https://github.com/awslabs/graphstorm/pull/66.
+        return 10000
```

Comment:
I think we need to provide more information in our argument doc about the selection of eval_batch_size.

Reply:
> 

Like the numbers or more reasons?

Reply:
Added.